plugins/pulp_rpm/plugins/importers/yum/utils.py

```diff
@@ -130,6 +130,8 @@ def fake_xml_element(repodata_snippet):
         # best second guess we have, and it will never fail due to the nature
         # of the encoding.
         codec = 'ISO-8859-1'
+    except UnicodeDecodeError:
+        repodata_snippet = repodata_snippet.decode("ISO-8859-1")
```

Comment:
This will result in the incorrect non-ASCII characters, because they will be decoded with "ISO-8859-1" and later encoded with "UTF-8".